insalan/tournament/serializers.py

```diff
@@ -188,6 +188,18 @@ def validate(self, data):
         if len(data.get("substitutes_name_in_games", [])) != len(data.get("get_substitutes_id", [])):
             raise serializers.ValidationError(_("Il manque des name_in_games de remplaçant⋅e⋅s"))
 
+        # Validate the name in game
+        for name in data.get("players_name_in_games", []):
```

Comment:
Why a plural here? Gramatically, `player_in_game_name` would be more correct

Reply:
It is plural because it's an array of name_in_game from every player joining the team but yeah, at least the s at the end of game is not needed.  
And this naming is not from today, I will change it but that require changing it in the front too.

Reply:
It happens to miss details yeah, sorry. If you want to keep a plural (that makes sense), it would be on `names` then, there are several names, not several games